deeppavlov/core/commands/train.py

```diff
@@ -194,6 +194,7 @@ def train_evaluate_model_from_config(config: [str, Path, dict], to_train: bool =
             }
 
             print(json.dumps(report, ensure_ascii=False))
+    return model
```

Comment:
If an early stopping occurred but the model was not tested, this will return not a best model.